Moonscraper Chart Editor/Assets/Scripts/Game/Charts/IO/Midi/MidReader.cs

```diff
@@ -7,6 +7,7 @@
 using UnityEngine;
 using NAudio.Midi;
 using MoonscraperEngine;
+using System.Collections.ObjectModel;
 
```

Comment:
Unnecessary, use IReadOnlyDictionary from Systems.Collections.Generic

Reply:
Gotcha, that's *much* saner to use lol